server/src/test/java/org/elasticsearch/cluster/metadata/MetaDataTests.java

```diff
@@ -418,20 +415,18 @@ public void testXContentWithIndexGraveyard() throws IOException {
         return new CoordinationMetaData.VotingConfiguration(Sets.newHashSet(generateRandomStringArray(randomInt(10), 20, false)));
     }
 
-    private Set<DiscoveryNode> randomDiscoveryNodeSet() {
+    private Set<VotingTombstone> randomVotingTombstones() {
         final int size = randomIntBetween(1, 10);
```

Comment:
We should probably test the empty case too.

Reply:
https://github.com/elastic/elasticsearch/pull/35832/commits/9c8d7a4bec7d69a3fbde83dd4e0daf1bee2efb9c